src/main/resources/liquibase/202205021439-add-billingProvider-and-billingAccountId-columns-to-host-tally-buckets.xml

```diff
@@ -0,0 +1,21 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="202205021439-1" author="ksynvrit">
+        <comment>Add column for billing account id to host tally buckets table</comment>
+        <addColumn tableName="host_tally_buckets">
+            <column name="billing_provider" type="VARCHAR(255)" />
+        </addColumn>
+    </changeSet>
+
+    <changeSet id="202205021439-2" author="ksynvrit">
+        <comment>Add column for billing account id to host tally buckets table</comment>
+        <addColumn tableName="host_tally_buckets">
+            <column name="billing_account_id" type="VARCHAR(255)" />
+        </addColumn>
```

Comment:
Both columns should have default `_ANY`, and have a non-null constraint. See `src/main/resources/liquibase/202006180903-add-usage-to-host.xml` for example.